python_transport/wirepas_gateway/transport_service.py

```diff
@@ -73,6 +197,24 @@ def __init__(self, settings, logger=None, **kwargs):
 
         self.logger.info("Gateway started with id: %s", self.gw_id)
 
+        self.monitoringThread = None
+        self.minimun_sink_cost = settings.bh_minimal_sink_cost
+
+        if settings.bh_max_buffered_packets > 0:
+            self.logger.info(" Black hole detection enabled: max_packets=%d packets, max_delay=%d",
```

Comment:
Have you considered another name instead of black hole detection? Maybe backend connection loss would be easier to get?